example/webpack4-babel/webpack.conf.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
 
   output: {
     path: path.resolve(__dirname, './dest'),
-    filename: 'bundle.js',
+    filename: '[name].js',
```

Comment:
this is more for consistency with the the other examples, but we could technically use `bundle.js` here since the issue is fixed in `karma-webpack` 4